Source/Core/Core/NetPlayServer.h

```diff
@@ -72,7 +75,9 @@ class NetPlayServer : public TraversalClientClient
 	};
 
 	void SendToClients(sf::Packet& packet, const PlayerId skip_pid = 0);
+	void SendToClients(std::unique_ptr<sf::Packet> packet, const PlayerId skip_pid = 0);
 	void Send(ENetPeer* socket, sf::Packet& packet);
+	void Send(ENetPeer* socket, sf::Packet* packet);
```

Comment:
Will packet be modified? If so, the one with a pointer is fine, else a constsf::Packet& is better. But please don't create both.